src/notifications.ts

```diff
@@ -28,25 +66,45 @@ export interface ViolatedLimit {
   current_value: number
 }
 
-export interface ProjectExceedingTierLimit {
+export type ProjectExceedingTierLimitData = {
+  name: NotificationName.ProjectExceedingTierLimit
```

Comment:
do we need this? since the name is already in the top level notifications object

Reply:
Doesn't having it here make it easier for you to use the union type within ts? I'm fine with removing it

Reply:
Though I need to add it to the rest of the types to support making it easier

Reply:
ahh okay no worries - lets keep this for now then 👍